debezium-connector-mysql/src/test/java/io/debezium/connector/mysql/MySqlAntlrDdlParserTest.java

```diff
@@ -2295,7 +2313,8 @@ public void shouldTreatPrimaryKeyColumnsImplicitlyAsNonNull() {
         assertThat(tableDef.columnWithName("id").hasDefaultValue()).isEqualTo(true);
         assertThat(tableDef.columnWithName("id").defaultValue()).isEqualTo(0);
 
-        ddl = "CREATE TABLE data(id INT DEFAULT 1, PRIMARY KEY (id))";
+        ddl = "DROP TABLE IF EXISTS data; " +
```

Comment:
Out of curiousity, how comes you added these DROP TABLE statements? Nothing speaking against it, just curious :)

Reply:
Well, because table with same name was already created in this same test, 10 lines above.
Options were to DROP it and CREATE, ot CREATE under another name.
I do not know much context of what exactly is being tested in this unit test, maybe creating it under new name would be actually better :)

Reply:
Ok thanks, makes sense.